### PR TITLE
261: Average Weekly Hours Card

### PR DESCRIPTION
**Pull request by ChazUK**
Adding the Average Weekly Hours Card.

@wozzer72 There's API calls on line 34 and 75 of `src/app/features/workers/average-weekly-hours/average-weekly-hours.component.ts`.

### CODE REVIEW

**Comment by ChazUK** [comment]
[Average weekly working hours #152](https://trello.com/c/QHFL4Qs9/261-average-weekly-working-hours-152)

File: src/app/features/workers/average-weekly-hours/average-weekly-hours.component.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+              <input type="radio" formControlName="weeklyHoursAverageKnown" class="form-control" value="Yes" for="weeklyHoursAverageKnown_yes">
+              <label for="weeklyHoursAverageKnown_yes">Yes</label>
+            </div>
+          </div>
+        </div>
+
+        <div *ngIf="form.get('weeklyHoursAverageKnown').value === 'Yes'" class="panel panel-border-narrow ml-0 mt-5 mb-5 mt-md-0">
+          <div class="form-field">
+            <label class="form-label">Average weekly hours
+              <input type="number" formControlName="weeklyHoursAverage" class="form-control" min="0" max="65" step="0.5" appNumberPositiveOnly [ngClass]="{'is-invalid': (form.get('weeklyHoursAverage').touched || form.get('weeklyHoursAverage').dirty) && !form.get('weeklyHoursAverage').valid}">
```
Comment (denny13) [inline src/app/features/workers/average-weekly-hours/average-weekly-hours.component.html]: This should be on multiple lines

File: src/app/features/workers/average-weekly-hours/average-weekly-hours.component.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+  weeklyHoursAverageKnownChangeHandler() {
+    this.form.controls['weeklyHoursAverage'].reset();
+  }
+
+  async submitHandler() {
+    try {
+      await this.saveHandler();
+
+      this.router.navigate([`/worker/payment-terms`]);
```
Comment (denny13) [inline src/app/features/workers/average-weekly-hours/average-weekly-hours.component.ts]: Try to use single quotes when you can

File: src/app/features/workers/average-weekly-hours/average-weekly-hours.component.html (hunk 1 above)
Comment (ChazUK) [reply src/app/features/workers/average-weekly-hours/average-weekly-hours.component.html]: This is another thing Prettier can take care of.
Do you want it on multiple lines due to the line length or having multiple attributes?

**Pull request merged**